css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
+	margin-top: 20px;
+	float: left;
+	margin-right: 150px;
+}
+
+header{
+	width: 100%;
+	height: 80px;
+	box-shadow: 0px 3px 10px grey;
+}
+
+footer{
+	margin-bottom: 0;
+}
+
+header span{
+	float: right;
+}
+
+.formated{
+	color: black;
+	font-size: 22px;
+}
+
+nav span{
+	margin-top: 30px;
+}
+
+span a{
+	color: black;
+	text-decoration: none;
+	margin-right: 60px;
+}
+
+.buttoni{
```

Comment:
I had button1 and button2. I wanted a class for both of them so I said button"i". I could have used just simple "button", right?